packages/react-components/react-tree/src/components/TreeItemActions/useTreeItemActionsStyles.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { makeStyles, mergeClasses, shorthands } from '@griffel/react';
+import type { TreeItemActionsSlots, TreeItemActionsState } from './TreeItemActions.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
+import { tokens } from '@fluentui/react-theme';
+
+export const treeItemActionsClassNames: SlotClassNames<TreeItemActionsSlots> = {
+  root: 'fui-TreeItemActions',
+};
+
+/**
+ * Styles for the action icon slot
+ */
+const useStyles = makeStyles({
+  base: {
+    display: 'none',
+    marginLeft: 'auto',
+    zIndex: 1,
+    ...shorthands.padding(0, tokens.spacingHorizontalS),
+    ...shorthands.gridArea('aside'),
+  },
+  open: {
+    display: 'flex',
+    position: 'relative',
+  },
+});
+
+/**
+ * Apply styling to the TreeItemActions slots based on the state
+ */
+export const useTreeItemActionsStyles_unstable = (state: TreeItemActionsState): TreeItemActionsState => {
+  const styles = useStyles();
+
+  state.root.className = mergeClasses(
+    treeItemActionsClassNames.root,
+    styles.base,
+    state.visible && styles.open,
```

Comment:
since the slot is conditionally rendered based on `visible`, do you really need the conditional style here?